server/routerlicious/packages/routerlicious-base/src/riddler/api.ts

```diff
@@ -142,11 +161,15 @@ export function create(
 		const tenantCustomData: ITenantCustomData = request.body.customData
 			? request.body.customData
 			: {};
+		const enableKeylessAccess = request.body.enableKeylessAccess
+			? request.body.enableKeylessAccess
+			: null;
 		const tenantP = manager.createTenant(
```

Comment:
This method should also have "enableKeyAcess". Both "enableKeylessAcess" and "enableKeyAcess" can be true, either of them can be true. If both of them are false, then it is an error condition.

Reply:
I do not think we need another flag - `enableKeyAccess`. Access via keys will continue to work even if `enableKeylessAccess` is set to true.